docs/reviewing.md

```diff
@@ -0,0 +1,37 @@
+# Reviewing policies
+
+The following policies are preferred during the review, but not mandatory:
+
+## Trailing white-spaces
+
+Avoid trailing white-space characters, if possible
+
+## Quotes
+
+If possible, try to avoid mixing single quotes (`'`) and double quotes (`"`) in python code (`conanfile.py`, `test_package/conanfile.py`). Consistency in preferred.
```

Comment:
```suggestion
If possible, try to avoid mixing single quotes (`'`) and double quotes (`"`) in python code (`conanfile.py`, `test_package/conanfile.py`). Consistency is preferred.
```